aiohttp/client_reqrep.py

```diff
@@ -527,6 +532,7 @@ def __init__(self, method, url, *,
         self._request_info = request_info
         self._timer = timer if timer is not None else TimerNoop()
         self._auto_decompress = auto_decompress
+        self._cache = {}
```

Comment:
This looks like redundant.

Reply:
Looks like no. All classes with @reify methods have this and AttributeError is raising without this line.

Reply:
Implicit magic. Worth to add notice about.

Reply:
Yes, it's required.

Reply:
I mean the attr is required, not a notice :)
But the notice makes no harm

Reply:
Oh, ok. Last time I saw `reify` in acttion, no special attrs were required.